pkg/cloud/schedules/schedules.go

```diff
@@ -177,12 +192,14 @@ func (l *LocalSchedulesService) Schedule(stream schedulespb.Schedules_ScheduleSe
 	case *schedulespb.RegistrationRequest_Every:
 		parts := strings.Split(strings.TrimSpace(t.Every.Rate), " ")
 		if len(parts) != 2 {
-			return fmt.Errorf("invalid schedule rate: %s", t.Every.Rate)
+			l.errorLogger(serviceName, fmt.Errorf("invalid rate: %s for schedule %s", t.Every.Rate, scheduleName))
+			return nil
 		}
 
 		initialRate, err := strconv.Atoi(parts[0])
 		if err != nil {
-			return fmt.Errorf("invalid schedule rate, must start with an integer")
+			l.errorLogger(serviceName, fmt.Errorf("invalid rate: %s for schedule %s", t.Every.Rate, scheduleName))
```

Comment:
the previos error description was more informative, I think we should also avoid any exact duplicate error description, since it makes it harder to narrow down where in the code they came from.